source/core/utilities.cc

```diff
@@ -258,7 +258,9 @@ fill_table_from_file(TableHandler     &table,
       myfile.close();
     }
   else
-    std::cout << "Unable to open file";
+    {
+      throw std::runtime_error("Unable to open file " + file_name + ".");
```

Comment:
Can you use the deal.II exception instead?
https://www.dealii.org/current/doxygen/deal.II/group__Exceptions.html#ga2f31207e832b937688f75c04e939cc21